stripper/ze_mapeadores_museum_v1.cfg

```diff
@@ -548,3 +548,166 @@ modify:
 		"OnUser2" "!activator,SetLocalOrigin,9590 11906 14023,10,1"
 	}
 }
+
+// Fix evading the spawn teleport.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"targetname" "MuseumSpawnMainTeleport"
+	}
+	replace:
+	{
+		"model" "*544"
+		"origin" "-14336 14336 13952"
+	}
+}
+
+// Kill the buttons in spawn when pressed.
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "secretbutton"
+	}
+	delete:
+	{
+		"OnPressed" "!selfKill0.11"
+	}
+	insert:
+	{
+		"OnPressed" "!self,Kill,,,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "secretpotato"
+	}
+	insert:
+	{
+		"OnPressed" "!self,Kill,,,1"
+	}
+}
+
+// Fix "func_breakable"s set to "None" material type producing wooden gibs.
+modify:
+{
+	match:
+	{
+		"classname" "func_breakable"
+		"material" "10"
+	}
+	replace:
+	{
+		"PerformanceMode" "1"
+	}
+}
+
+// Kill the ending button on "SSS-Lite" when it's pressed.
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "hich_sss_endbutton"
+	}
+	insert:
+	{
+		"OnPressed" "!self,Kill,,,1"
+	}
+}
+
+// Fix "SSS-Lite" ending teleport using incorrect name for its filter.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"targetname" "hich_sss_endtp_ct"
+	}
+	replace:
+	{
+		"filtername" "hich_filter_allow_ct"
+	}
+}
+
+// Fix "GOLEVATOR" elevator getting people stuck on spawn and fix delaying.
+add:
+{
+	"classname" "trigger_once"
```

Comment:
I assume you want this to be trigger_multiple?

Reply:
trigger_once is fine since players has to only touch it for 1 frame and trigger_once has delay before death and OnStartTouch can fire multiple time during this.

Reply:
My bad, I assumed you were trying to fix people blocking the elevator on the way up. I assume you may want to look into that though, `blockdamage` seems to not have an effect for whatever reason.

Reply:
That was caused by map giving players godmode using a filter, I changed that filter to a team filter now at the bottom of the config, it is also auto-removed after 30 seconds of platform spawn to prevent delaying.